main.py

```diff
@@ -60,5 +60,28 @@ def submitNote():
         # Unsure if this would work. Directory is referenced, but seems redundant.
         return (qnTitle + '\n' + qnContent+ dateMessage)
 
+    @app.route('/file_read', methods=['GET'])
+    def dataPull():
+        data = {'Title': 'a', 'Contents': 'b', 'Creation_Date': 'c', 'Last_Modified': 'd'}
+        basepath = os.path.realpath('./submitNote')
+        dataArray = []
+        for file in os.listdir(basepath):
+            filetitle = str(file)
+            filepath = os.path.join(basepath, file)
+            creatstamp= os.path.getctime(filepath)
+            timestamp = os.path.getmtime(filepath)
+            data.update(Last_Modified=time.ctime(timestamp))
+            data.update(Creation_Date=time.ctime(creatstamp))
+            data.update(Title=filetitle)
+            with open(filepath, 'r') as f:
+                mainbody = (f.read())
+            data.update(Contents=mainbody)
+            dataArray.append(data.copy())
+        jsonArray = json.dumps(dataArray)
+        return jsonArray
+<<<<<<< Updated upstream
```

Comment:
This Updated upstream means you have a local stash that you applied to the branch that's causing a minor conflict, just remove this and should be good.